commands/pipelines/connect.js

```diff
@@ -0,0 +1,62 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+const api = require('../../lib/api')
+const KolkrabbiAPI = require('../../lib/kolkrabbi-api')
+const GitHubAPI = require('../../lib/github-api')
+
+const Validate = require('./setup/validate')
+const getGitHubToken = require('./setup/getGitHubToken')
+const getNameAndRepo = require('./setup/getNameAndRepo')
+const getRepo = require('./setup/getRepo')
+
+module.exports = {
+  topic: 'pipelines',
+  command: 'connect',
+  description: 'connect a github repo to an existing pipeline',
+
+  help: `Example:
+
+    $ heroku pipelines:connect example githuborg/reponame
+    Configuring pipeline... done`,
+  needsApp: false,
+  needsAuth: true,
+  wantsOrg: false,
+  args: [{
+      name: 'name',
+      description: 'name of pipeline',
+      optional: true
+    },
+    {
+      name: 'repo',
```

Comment:
I wonder if we should make this argument more explicit via a flag.

e.g.:

```
$ heroku pipelines:connect PIPELINE --to user/repo
$ heroku pipelines:connect PIPELINE --repo user/repo
```

I'm not entirely convinced by this though. It's just a thought that I've always found that make it easier to remember, so users don't have to remember the arguments order. I'm aware this suggestion is actually changing the design other pipelines commands, so this is probably not relevant to this PR.

Reply:
I like your proposal. I personally prefer arguments.